data/common.yaml

```diff
@@ -20,6 +23,13 @@ mcollective::server_config:
   plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
   registerinterval: 10
   registration: "Choria"
+  logfile: "/var/log/puppetlabs/mcollective.log"
```

Comment:
> The more mundane things they will not goes into these hashes.

If I understand the hash correctly ([it is added last](https://github.com/choria-io/puppet-mcollective/blob/master/manifests/config.pp#L29https://github.com/choria-io/puppet-mcollective/blob/master/manifests/config.pp#L29)), nothing stops me from putting my entire config, including `logfile`.

Though this might be an edge case, how would you like me do proceed with merging?